packages/xo-web/src/index.scss

```diff
@@ -213,6 +213,7 @@ $select-input-height: 40px; // Bootstrap input height
   border: 1px solid black;
   margin: 5px 10px;
   width: 250px;
+  z-index: 3;
```

Comment:
Can you explain why this is necessary?

Why does the search box was above?

Reply:
[input-group](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/sorted-table/index.js#L82) , [form-control for input ](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/sorted-table/index.js#L104) and [input-group-btn](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/sorted-table/index.js#L118) have caused this. 
Without these styles, the notification isn't hidden.
I searched in the bootstrap code, I found that  `z-index: 3` for focused [form-control](https://github.com/twbs/bootstrap/blob/master/scss/_input-group.scss#L30TOL34) so I fixed the prblem by adding `z-index: 3`.

I'm still looking for the real problem  ^^

Reply:
This is a good explanation which should, IMHO, be in the code as a comment.

Reply:
I don't know how the pop-up of the notification trigger the focus event.

Reply:
@badrAZ  I fixed the comment and I think it's clear to answer your question. ^_^

Reply:
I'm afraid to fix something specific to a UI in a generic CSS. What if a component has a z-index > 3 ? I suggest to fix it on the concerned select.